server/database/queries/updateTask.js

```diff
@@ -0,0 +1,12 @@
+const connect = require('../config/connection');
+
+const updateTask = (task, scrumId, taskId) => {
+  const taskToUpdate = Object.keys(task).map(value => task[value]);
```

Comment:
you could do it this way: 
```javascript
const { action_type, status, modules ... } = task
```
Inside the `updatedTask object`:
```javascript
 values: [action_type, status, modules..., scrumId, taskId]
```

You cant' expect how the object keys will be organized.